sdk/python/feast/client.py

```diff
@@ -470,35 +475,50 @@ def get_online_features(
         )  # type: GetOnlineFeaturesResponse
 
     def ingest(
-        self,
-        feature_set: Union[str, FeatureSet],
-        source: Union[pd.DataFrame, str],
-        version: int = None,
-        force_update: bool = False,
-        max_workers: int = CPU_COUNT,
-        disable_progress_bar: bool = False,
-        chunk_size: int = 5000,
-        timeout: int = None,
-    ):
+            self,
+            feature_set: Union[str, FeatureSet],
+            source: Union[pd.DataFrame, str],
+            chunk_size: int = 10000,
+            version: int = None,
+            force_update: bool = False,
+            max_workers: int = CPU_COUNT - 1,
+            disable_progress_bar: bool = False,
+    ) -> None:
         """
         Loads feature data into Feast for a specific feature set.
 
         Args:
-            feature_set: Name of feature set or a feature set object
-            source: Either a file path or Pandas Dataframe to ingest into Feast
+            feature_set (typing.Union[str, FeatureSet]):
+                Feature set object or the string name of the feature set
+                (without a version).
+
+            source (typing.Union[pd.DataFrame, str]):
+                Either a file path or Pandas Dataframe to ingest into Feast
                 Files that are currently supported:
-                * parquet
-                * csv
-                * json
-            version: Feature set version
-            force_update: Automatically update feature set based on source data
-                prior to ingesting. This will also register changes to Feast
-            max_workers: Number of worker processes to use to encode values
-            disable_progress_bar: Disable printing of progress statistics
-            chunk_size: Maximum amount of rows to load into memory and ingest at
-                a time
-            timeout: Seconds to wait before ingestion times out
+                    * parquet
+                    * csv
+                    * json
+
+            chunk_size (int):
+                Amount of rows to load and ingest at a time.
+
+            version (int):
+                Feature set version.
+
+            force_update (bool):
+                Automatically update feature set based on.
```

Comment:
based on...?

Reply:
My bad, I'll amend this in the next commit.